lib/logger_json.ex

```diff
@@ -56,6 +56,9 @@ defmodule LoggerJSON do
 
   Some formatters require additional configuration options. Here are the options that are common for each formatter:
 
+    * `:encoder_opts` - options to be passed directly to the JSON encoder. This allows you to customize the behavior of the JSON
+    encoder. By default, no options are passed to the encoder.
```

Comment:
@AndrewDryga not sure if we want to point to Jason's encoder opts here. The only other place I could see it making sense is in `LoggerJSON.Formatter` but that doesn't have any documentation at the moment.

Reply:
@anthonator I think this is a good place as all adapters refer to this docs for all shared options.

Reply:
Resolved by [c00876a](https://github.com/Nebo15/logger_json/pull/130/commits/c00876af405606a3f06290e6fa56cac272c2ba63)